### DISCUSSION TITLE
GatewayBase abstract methods usage

### DISCUSSION THREAD

**Discussion by MDP66** [Sina-Soltani/Parbad#177] [Q&A]
Hello there,

The Documents for new gateways is not sufficient.I need more information about what is the usage of three abstract method on `GatewayBase` class.

Please provide more information.

**Comment by Sina-Soltani** [comment]
Hi,
I think their names tell everything clearly.
But I just updated the document.
[What do Request, Verify and Refund do?](https://github.com/Sina-Soltani/Parbad/wiki/Developing-a-new-Gateway#what-do-request-verify-and-refund-do)

**Reply by MDP66** [reply]
Thank you sir.
